community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/logical/QueryPlanner.scala

```diff
@@ -28,9 +28,9 @@ trait QueryPlanner {
   def plan(plannerQuery: UnionQuery)(implicit context: LogicalPlanningContext): LogicalPlan
```

Comment:
Well, not really from the branch name or the PR name, but me and Andrés felt that it was so small it might as well be part of this PR since it was also pretty small and isolated. Do you think we should make a dedicated PR for this?